src/components/Navbar.tsx

```diff
@@ -1,38 +1,150 @@
 import Navbar from "react-bootstrap/Navbar";
-import Form from "react-bootstrap/Form";
+import { useEffect } from "react";
+import { useState } from "react";
+import "../css/App.css";
+import { NavLink } from "react-router-dom";
 import Button from "react-bootstrap/Button";
-import InputGroup from "react-bootstrap/InputGroup";
-import Row from "react-bootstrap/Row";
-import Col from "react-bootstrap/Col";
+import { FaBars } from "react-icons/fa";
+import { useNavigate } from "react-router-dom";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
bootstrap  should  not be imported here  @Rishab-innovative it should be global imported once